.scripts/torus.gradle

```diff
@@ -4,9 +4,11 @@ android {
 
         debug {
             resValue "string", "torusVerifier", "${torusProperties.getOrThrow('TORUS_VERIFIER_DEBUG')}"
+            resValue "string", "torusSubVerifier", "${torusProperties.getOrThrow('TORUS_SUB_VERIFIER_FEATURE')}"
```

Comment:
I am not sure if we need subverifier for debug builds. 
There is different verifier

Reply:
i'll get you an update on the changes with verifiers